cloudant-client/src/main/java/com/cloudant/client/api/ClientBuilder.java

```diff
@@ -607,4 +610,62 @@ public ClientBuilder readTimeout(long readTimeout, TimeUnit readTimeoutUnit) {
         return this;
     }
 
+    /**
+     * Sets Cloudant client credentials by inspecting the VCAP_SERVICES Cloud Foundry environment
+     * variable. The VCAP_SERVICES variable contains connection information to access a Bluemix
+     * service instance.
+     *
+     * @return a new ClientBuilder for the account
+     */
+    public static ClientBuilder bluemix() {
+        return ClientBuilder.bluemix(null);
+    }
+
+    /**
+     * Sets Cloudant client credentials by inspecting the VCAP_SERVICES Cloud Foundry environment
+     * variable. The VCAP_SERVICES variable contains connection information to access a Bluemix
+     * service instance.
+     * <P>
+     * Note: Specifying a service name is only required when multiple Cloudant services are
+     * available.
+     * </P>
+     *
+     * @param serviceName name of Bluemix service instance
+     * @return a new ClientBuilder for the account
+     */
+    public static ClientBuilder bluemix(String serviceName) {
+        Gson gson = new GsonBuilder().create();
+
+        String vcap_env = System.getenv("VCAP_SERVICES");
+        if (vcap_env == null) {
+            throw new IllegalArgumentException("VCAP_SERVICES is not present.");
```

Comment:
Rhys and I talked about using `IllegalStateException` for these. It does make more sense I think. I'll update.